public/app/features/admin/AdminListOrgsPage.tsx

```diff
@@ -29,6 +29,22 @@ export const AdminListOrgsPages: FC = () => {
       <Page.Contents>
         <>
           <div className="page-action-bar">
+            <InfoBox branded>
+              <VerticalGroup spacing="xs">
+                <Badge text="Info" color="blue" icon="info-circle" />
```

Comment:
```suggestion
                <Badge text="Warning" color="blue" icon="info-circle" />
```
Im not sure if warnings exists but I think the "level" should be higher.

Reply:
removed the badge, the warnings and red color look too strong